compiler/test/runner.re

```diff
@@ -98,6 +106,12 @@ let run = (~num_pages=?, file) => {
     | _ => failwith("process did not exit properly")
     };
 
+  let out =
+    if (timed_out) {
+      "Timed out!\n" ++ out;
```

Comment:
This is just to make the test fail. It was simpler than wiring up the `expect` values, and it has the added benefit of still displaying whatever output _did_ manage to get printed.